app/Eloquent/Vote.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Eloquent;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Vote extends Model
+{
+    protected $fillable = [
+    	'status',
+    	'user_id',
+    	'review_id',
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Eloquent/Vote.php</br>Spaces must be used to indent lines; tabs are not allowed